nginx/tests/nginx_vts/nginx.conf

```diff
@@ -0,0 +1,44 @@
+worker_processes  1;
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+    vhost_traffic_status_zone;
+
+    sendfile        on;
+    keepalive_timeout  65;
+
+    # HTTP
+    server {
+        listen       80;
+        server_name  localhost;
+
+        location / {
+            root   /usr/share/nginx/html;
+            index  index.html index.htm;
+        }
+
+        location /nginx_status {
+          stub_status on;
+          allow all;
+        }
+
+        location /vts_status {
```

Comment:
this is gonna need some additional docs I think, probably largely out of scope for this but can you add that to the example file?

Reply:
What would be the target for such docs in your opinion? Devs or users?

Reply:
Users, I would think.

Reply:
uhm the option is documented in the config file, we can assume if someone wants to monitor the VTS module they know how to configure it